### ISSUE TITLE
Avoid the autoloading in laminas/laminas-zendframework-bridge

### ISSUE THREAD

**Issue by Danack** [laminas/laminas-escaper#11]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | yes
| BC Break    | maybe

#### Summary

I'm in the process of moving from using zend-escaper to this fork.

I noticed that through the require of zendframework-bridge a custom autoloader is added. Is there, or could there be, a way of avoiding this?

I'm using this library separately from zendframework (name through https://github.com/danack/esprintf) and having a simple library require having a custom autoloader adds a bit of overhead and complexity that doesn't feel great.

I have no idea how this could be done while still meeting your requirements.

**Comment by Ocramius** [comment]
The bridge component can be dropped in a new major: it was introduced to ease migration from `Zend\` to `Laminas\` without having to bump major.

Feel free to send a patch to remove it in `3.0.x` (I'll change the PR target afterwards)

**Comment by froschdesign** [comment]
@Danack 
Another option is to [avoid the installation](https://getcomposer.org/doc/04-schema.md#replace) of laminas-zendframework-bridge via Composer in your project / application:

```json
"replace": {
    "laminas/laminas-zendframework-bridge": "*"
},
```

[But this can also lead to side effects with 3rd-party libraries.](https://github.com/laminas/laminas-httphandlerrunner/pull/9#issuecomment-829023884)

**Comment by boesing** [comment]
I would not encurage adding this to any library. 
That should be done by the root project rather than libs.

If a library uses this replacement, upstream projects will end up having potential `class not found` issues if these depend on 3rd-party libs which did not yet migrate.

**Comment by froschdesign** [comment]
Fixed with #27 and released with [2.9.0](https://github.com/laminas/laminas-escaper/releases/tag/2.9.0).
